software/ChromeAndPuppeteer.md

```diff
@@ -0,0 +1,142 @@
+# Headless website testing with Chrome and Puppeteer on Graviton.
+
+Chrome is a popular reference browser for web site unit testing on EC2 x86 instances. In the samer manner it can be used on EC2 Graviton instances.
+It is open source in its Chromium incarnation, supports 'headless' mode and can simulate user actions via Javascript.
+The Chrome web site [mentions Puppeteer](https://developer.chrome.com/blog/headless-chrome/#using-programmatically-node) and it is used here to show a specific
+example of headless website testing on Graviton.
+“[Puppeteer](https://pptr.dev/) is a Node.js library which provides a high-level API to control Chrome/Chromium over the DevTools Protocol. 
+Puppeteer runs in headless mode by default, but can be configured to run in full ("headful") Chrome/Chromium."
+It can serve as a replacement for the previously very popular Phantomjs.
+“PhantomJS (phantomjs.org) is a headless WebKit scriptable with JavaScript. The latest stable release is version 2.1.
+Important: PhantomJS development is suspended until further notice (see #15344 for more details).“
+The APIs are different, so code targetting PhantomJS has to be rewritten when moving to Puppeteer (see Appendix).
+Puppeteer is open source and has 466 contributors and 364k users and thus is likely to be supported for some time.
```

Comment:
I'd just say "many contributors and users" because the numbers will change constantly.